src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -80,7 +81,20 @@ const FormDefinitionDetailPage = () => {
 
   const isPublished =
     formIdentifier.displayVersion.status === FormStatus.Published;
+  const publishedBy = isPublished
```

Comment:
This might be contentious - I'm not 100% sure about the assumption that the last person to update a published form was indeed the person to publish it. (We get the person from the [audit history](https://github.com/greenriver/hmis-warehouse/blob/11be216286e6b81d4e1367813d8ea53bb3a69b59/drivers/hmis/app/graphql/types/forms/form_definition.rb#L117))

I feel like that assumption _should_ be true because you can't edit a published form... but maybe making the assumption is brittle?

Reply:
Yep thats how we are using those fields  in the version history. There is no other mechanism for updating forms. I agree there could be drawbacks (for example, if we need to update forms in a migration and accidentally set updated_at in the process). Such a change could be rolled back though since we use paper trail. I don't think it makes sense to change this pattern at this stage